source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -0,0 +1,160 @@
+url for a given shop link in the following table:
+
+##### Valid webhook url
+
+Shop URL | Webhook URL
+-------- | ----------
+`https://shop.gr` | `https://shop.gr/webhook_url`
+
+## Webhook events
+
+A webhook request is sent to the predefined webhook url once an event has been triggered.
+As of right now, the only supported webhook event is the placement of a
+<strong>new Smart Cart order</strong>. We provide an overview of the request
+headers and payload to be sent by our side to the webhook url.
+
+### New order
+
+A <strong>new order</strong> webhook request is triggered upon the placement of a new order.
+Any failed request should be retried in a later time. It should be expected that a request
+concerning the same order would be performed up to 3 times in total, every 15 minutes.
+We also provide the possibility of manually sending / retrying a new order webhook request
+from within the order page, however the expected request would be the same in all the above
+scenarios.
+
+##### Request
+<pre class="terminal">
+POST {webhook_url}
+</pre>
+
+##### Expected request response
+
+<pre class="terminal">
+200
+</pre>
+
+##### Request headers
+
+Header         | Value
+-------------- | -----
+`Content-Type` | `application/json: charset=utf-8`
+`User-Agent`   | `Skroutz OrderNotifier v1`
+
+##### Request payload
+
+Name         | Type   | Description
+:------------: | :------: | :----------------:
+`event_type` | String | Order event type (value: `"new_order"`)
+`order`      | Object | [Order details](#order-object)
+
+#### Order object
+
+Name         | Type   | Description
+:------------: | :-------: | :---------------:
+`code`       | String | Order code
+`comments`   | String | Order comments
+`line_items` | Array | [Order line items (products)](#order-line-items-array)
+`created_at` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order creation date
+`expires_at` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order expiration date
+`dispatch_until` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order maximum dispatch date
+
+#### Order line items array
+
+Name                              | Type      | Description
+:-------------------------------: | :-------: | :----------------------------------------------:
+`line_items[_].shop_uid`        | String  | Item id in shop
+`line_items[_].product_name`    | String  | Item product name
+`line_items[_].quantity`        | Integer | Item quantity
+`line_items[_].size`            | Object  | [Item size values (optional)](#line-item-size)
+`line_items[_].unit_price`      | Double  | Price per item in euros
+`line_items[_].total_price`     | Double  | Total item price in euros: `unit_price` * `quantity`
+
+#### Line item size
+
+Name                            | Type    | Description
+:-------------------------------: | :-------: | :------------------------------------------:
+`line_items[_].size.unit`       | String  | Item size unit
```

Comment:
We should note that `unit` refers to a label about the size or even list the possible values `Νούμερο / Μέγεθος / Ηλικία`. (Not to be confused with actual size units... IS, WL, UK etc)

Reply:
After renaming the `unit` to `label` on the payload, I believe that we could simply mention the three possible vaues. WDYT?